src/core.jl

```diff
@@ -224,6 +224,7 @@ by `model.batch_size`.)
 
 """
 function collate(model, X, y)
+    y = y isa Matrix ? Tables.table(y) : y
```

Comment:
I'm trying to understand why this fix was needed. The `nrows` function is supposed to work for matrices as well as tables:

```julia
using MLJBase
julia> y = rand(2, 6)
julia> nrows(y)
2

julia> nrows(y')
6
```
And line 230 below should already take care of the conversion of `y` to a matrix, no?

Reply:
This was giving me an error previously, because the `nrows` method defined in the same file expects a table (or else it throws an `ArgumentError`. I've adjusted that and tests are passing.